lib/core/config.js

```diff
@@ -148,6 +148,7 @@ Config.prototype._mergeConfig = function(configFilePath, defaultConfig, env, ena
 
   let config;
   if (fs.existsSync(configFilePath + '.js')) {
```

Comment:
this could be replaced with an `else` for line 141 🙃

Reply:
I think these 2 blocks have enough different roles to justify being separate, albeit the method could be refactored, but prefer to do that in a different PR